test/forge/FeeTest.sol

```diff
@@ -40,6 +40,18 @@ contract FeeTest is IntegrationTest {
         _setCap(allMarkets[0], CAP);
     }
 
+    function testSetFee(uint256 fee) public {
+        vm.assume(fee != vault.fee());
+        fee = bound(fee, 0, ConstantsLib.MAX_FEE);
```

Comment:
```suggestion
        fee = bound(fee, 0, ConstantsLib.MAX_FEE);

        vm.assume(fee != FEE);
```

Otherwise it's not enough, `fee` can still be `FEE + k * ConstantsLib.MAX_FEE`

Reply:
lol i did this change in local but forgot to git add them... So yes 100% agree